recipes/amplisim/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.2.0" %}
+{% set sha256 = "6eb3aa9ef7dd803a7bb7675a0a78dbb31a6cecce6b8d8cc2e33c48e1560bbea2" %}
+
+package:
+  name: amplisim
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/rki-mf1/amplisim/archive/v{{ version }}.tar.gz
+  git_tag: v{{ version }}
```

Comment:
close, but not quite :)

See here: https://github.com/bioconda/bioconda-recipes/blob/c75c09453f56ba77e3ed01436ddb2e5852ceac4f/recipes/seqhax/meta.yaml#L8-L10